docs/source/workflows.rst

```diff
@@ -152,6 +153,10 @@ What it can not do:
 Note that if the ``pressure_list`` value is provided, the other pressure inputs are neglected and the automatic pressure
 selection of the work chain is skipped.
 
+Note that you can scale the probe radius to empirically account for some framework flexibility and avoid overblocking. 
+Setting ``zeopp_probe_scaling`` to zero (or a small value) basically correspond to skip the permeability check and 
+skip the calculation of blocking spheres.
```

Comment:
```suggestion
skips the calculation of blocking spheres.
```

Reply:
Is this true?
Will setting it to a small non-zero value skip the pore blocking computation?

Reply:
No, because I didn't implement the logic to skip it but in practice (basically) it is skipping it because zeopp will not find blocking spheres. LEt's not be too picky it is a negligible waste of efficiency